qiita_pet/templates/software.html

```diff
@@ -0,0 +1,69 @@
+{% extends sitebase.html %}
+{% block head %}
+{% from future.utils import viewitems %}
+{% from qiita_core.qiita_settings import qiita_config %}
+
+<script type="text/javascript">
+$(document).ready(function() {
+    $('#error-table').dataTable({"order": [[1, "asc"]]});
+    $("#waiting").hide();
+} );
+</script>
+
+{% end %}
+
+{% block content %}
+  {% if software %}
+    <table class="table-bordered" width="100%">
+      <thead>
+        <tr>
+          <th>Software</th>
+          <th>Active</th>
+          <th>Commands</th>
+        </tr>
+      </thead>
+      <tbody>
+      {% for s in software %}
+        <tr>
+          <td>{{s.name}}, {{s.version}}</td>
+          <td>
+            {% if s.active %}
+              <span class="glyphicon glyphicon-thumbs-up" style="color:green"></span>
+            {% else %}
+              <span class="glyphicon glyphicon-thumbs-down" style="color:red"></span>
+            {% end %}
+          </td>
+          <td>
+            <table class="display table-bordered table-hover" width="100%">
+              <thead>
+                <tr>
+                  <th>Command</th>
+                  <th>Active</th>
+                </tr>
+              </thead>
+              <tbody>
+                {% for c in s.commands %}
+                  <tr>
+                    <td>{{c.name}}</td>
+                    <td>
+                      {% if c.active %}
+                        <span class="glyphicon glyphicon-thumbs-up" style="color:green"></span>
+                      {% else %}
+                        <span class="glyphicon glyphicon-thumbs-down" style="color:red"></span>
+                      {% end %}
+                    </td>
+                  </tr>
+                {% end %}
+              </tbody>
+            </table>
+          </td>
+        </tr>
+      {% end %}
+      </tbody>
+    </table>
+  {% else %}
+      <div id="jumbotron" class="jumbotron">
+          <h1><span class="glyphicon glyphicon-thumbs-down"></span> There is no plugin software in this system </h1>
```

Comment:
What about: `There are no plugins in this system`.

Reply:
k